docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -29,113 +29,73 @@ components:
 
-* `discoverable`: If an endpoint is discoverable, it means that it can be accessed using its name as the host name within the workspace containers (in the Kubernetes parlance, a Service is created for it with the provided name).
-
-* `public`: The endpoint will also be accessible outside of the workspace: such endpoint can be accessed from the {prod-short} user interface and is always exposed on port `80` or `443` (depending on whether `tls` is enabled in {prod-short}).
+* `exposure`: The endpoint will also be accessible outside of the workspace: such an endpoint can be accessed from the {prod-short} user interface and is always exposed on port `80` or `443` depending on whether `tls` is enabled in {prod-short}.
 
 * `protocol`: For public endpoints the protocol is a hint to the UI on how to construct the URL for the endpoint access. Typical values are `http`, `https`, `ws`, `wss`.
 
-* `secure`: A boolean (defaulting to `false`) specifying whether the endpoint is put behind a JWT proxy requiring a JWT workspace token to grant access. The JWT proxy is deployed in the same Pod as the server and assumes the server listens solely on the local loopback interface, such as `127.0.0.1`.
+* `secure`: A boolean defaulting to `false` that specifies whether the endpoint is put behind a JWT proxy requiring a JWT workspace token to grant access. The JWT proxy is deployed in the same Pod as the server and assumes the server listens solely on the local loopback interface, such as `127.0.0.1`.
 +
 WARNING: Listening on any other interface than the local loopback poses a security risk because such server is accessible without the JWT authentication within the cluster network on the corresponding IP addresses.
 
 * `path`: The URL of the endpoint.
 
-* `unsecuredPaths`: A comma-separated list of endpoint paths that are to stay unsecured even if the `secure` attribute is set to `true`.
-
-* `cookiesAuthEnabled`: When set to `true` (the default is `false`), the JWT workspace token is automatically fetched and included in a workspace-specific cookie to allow requests to pass through the JWT proxy.
-+
-WARNING: This setting potentially allows a link:https://en.wikipedia.org/wiki/Cross-site_request_forgery[CSRF] attack when used in conjunction with a server using POST requests.
+When starting a new server within a component, {prod-short} autodetects this, and the UI offers to automatically expose this port as an `exposure` port. This exposure is useful for debugging a web application. It is impossible to do this for servers that autostart with the container, for example, a database server. For such components, specify the endpoints explicitly.
```

Comment:
Checking this section against what it's replacing, it seems that `public` is a possible value for `exposure`, but simply replacing the word `public` with `exposure` does not convey how the `exposure` property actually works. It's not a boolean value, it's a list of specific exposure types as far as I can tell.

Reply:
Will address this concern in a PR for the newly filed issue, #393: https://github.com/devfile/api/issues/393